internal/cli/atlas/privateendpoints/gcp/interfaces/describe.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interfaces
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	privateEndpointServiceID string
+	privateEndpointGroupID   string
+	store                    store.InterfaceEndpointDescriber
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var describeTemplate = `ENDPOINT	STATUS	DELETE REQUESTED
+{{.EndpointGroupName}}	{{.Status}}	{{.DeleteRequested}}
+`
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.InterfaceEndpoint(opts.ConfigProjectID(), provider, opts.privateEndpointServiceID, opts.privateEndpointGroupID)
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas privateEndpoint(s) gcp interface(s) describe <endpointGroupId> --endpointServiceId endpointServiceId [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <endpointGroupId>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint interface for your project.",
+		Annotations: map[string]string{
+			"args":                "endpointGroupId",
+			"requiredArgs":        "endpointGroupId",
+			"endpointGroupIdDesc": "Unique identifier for the endpoint group.",
```

Comment:
Please see this [description](https://github.com/mongodb/mongocli/pull/921/files#diff-12aba3cdbb338fc867f81690686551f33a06c96c549f29f1b859637f8e26bf82R59-R60) -- shouldn't they all match?

Reply:
No, they are not the same and should not match, Julia.
The arg you are referring to is a private endpoint ID that is specific to AWS.
Here we are using something different: an endpoint group ID that is specific to GCP.

Reply:
@johansteyn if so, shouldn't we still use the same style for the description, and indicate the number or chars?

Reply:
@JuliaMongo Adding to what I said: when creating a GCP endpoint in the Atlas UI, there are no restrictions on the "Private Service Connect Endpoint Prefix" field, other than that it be unique. This field is what is used as the "endpointGroupId".

Reply:
I should have checked the docs for this myself and thank you for your patience. Agreed now!